Runtime/Behaviours/CinemachineConfiner.cs

```diff
@@ -82,6 +82,7 @@ class VcamExtraState
         {
             public Vector3 m_previousDisplacement;
             public float confinerDisplacement;
+            public bool isFramingTransposer;
```

Comment:
I would call this applyAfterAim - it's not necessarily the framing transposer, there could be another component with that property